care/facility/models/patient_base.py

```diff
@@ -53,11 +53,19 @@ def reverse_choices(choices):
     "Diabetes": 2,
     "Heart Disease": 3,
     "HyperTension": 4,
-    "Kidney Diseases": 5,
-    "Lung Diseases/Asthma": 6,
+    "Chronic Renal Disease": 5,
+    "Asthma": 6,
```

Comment:
can we change `Lung Diseases/Asthma` to `Other Chronic Lung Diseases` and make asthma a separate option?

cc: @aparnacoronasafe